### ISSUE TITLE
Invalid ARIA attribute value: aria-owns="results-desktop"

### ISSUE THREAD

**Issue by mgifford** [gctools-outilsgc/aurora-website#292]
This is a WCAG violation. 

<img width="721" alt="Screen capture of Accessibility Insight's error" src="https://user-images.githubusercontent.com/116832/62711367-efd6b680-b9c6-11e9-8696-656b3e2c2c0e.png">

What is this supposed to do?

**Comment by s-duffey** [comment]
@ethanWallace could you check this out?

**Comment by ethanWallace** [comment]
The problem comes from the dom element with the id results-desktop not being on the page until a search query has been entered into the search box. Thanks for pointing this out.
